src/table/thead.tsx

```diff
@@ -110,24 +124,32 @@ const Thead = React.forwardRef(
           )}
           {selectionType === 'single' && (
             <th
-              className={clsx(headerCellClass, selectionCellClass, hidden && headerCellStyles['header-cell-hidden'])}
+              className={clsx(
+                headerCellClass,
+                selectionCellClass,
+                hidden && headerCellStyles['header-cell-hidden'],
+                !shouldDisableStickyColumns && hasFirstStickyColumns && headerCellStyles['sticky-cell']
+              )}
+              style={{ left: `${cellOffsets?.first[0]}px` }}
```

Comment:
Why are you using a string value here, and a number value above?

Reply:
Not intentional - fixed!